src/foremast/awslambda/awslambda.py

```diff
@@ -164,7 +164,7 @@ def update_alias(self):
             LOG.info("Alias update failed. Retrying...")
             raise
 
-    @retries(max_attempts=3, wait=1, exceptions=(SystemExit))
+    @retries(max_attempts=10, wait=3, exceptions=(SystemExit))
```

Comment:
I think the max_attempts is off and should be reverted back to `3` attempts for now, but the wait should be increased to 3 as you have done. 1 second delay is potentially too aggressive.

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request